desktop/mac/index.md

```diff
@@ -444,6 +440,35 @@ After you have enabled two-factor authentication:
 
 After you have successfully authenticated, you can access your organizations and repositories directly from the Docker Desktop menu.
 
+## Pause/Resume
+
+Starting with the Docker Desktop 4.2 release, you can pause your Docker Desktop session when you are not actively using it and save CPU resources on your machine. When you pause Docker Desktop, the Linux VM running Docker Engine is paused, the current state of all your containers are saved in memory, and all processes are frozen. This reduces the CPU usage and helps you retain a longer battery life on your laptop. You can resume Docker Desktop when you want by clicking the Resume option.
+
+From the Docker menu, select ![whale menu](images/whale-x.png){: .inline} > **Pause** to pause Docker Desktop.
+
+![Docker context menu](images/menu/prefs.png){:width="250px"}
+
+Docker Desktop now displays the paused status on the Docker menu and on the  **Containers / Apps**, **Images**, **Volumes**, and **Dev Environment** screens on the Docker Dashboard. You can still access the **Preferences** and the **Troubleshoot** menu from the Dashboard when you've paused Docker Desktop.
```

Comment:
nit: do we want a generic form instead of the list of screens? Otherwise we’ll have to remember we need to update that list when we add new ones?
```suggestion
Docker Desktop now displays the paused status on the Docker menu and on all screens of the Docker Dashboard. You can still access the **Preferences** and the **Troubleshoot** menu from the Dashboard when you've paused Docker Desktop.
```
Well, it’s not quite all I suppose, as the Settings and Troubleshoot are still accessible…

Reply:
The generic form sounds good. There's call out on accessing Settings and Troubleshoot even when Docker Desktop is paused.

Reply:
Updated